utils/v2tofhirr4/utilities.bal

```diff
@@ -620,3 +606,65 @@ function transformToFhir(hl7:Message message) returns json|error {
     return getOperationOutcome(string `Unsupported message: ${message.name}`);
 }
 
+# Union type for holding patient name for all supported hl7 versions.
+public type Pid5 hl7v23:XPN[]|hl7v231:XPN[]|hl7v24:XPN[]|hl7v25:XPN[]|hl7v251:XPN[]|hl7v26:XPN[]|hl7v27:XPN[]|hl7v28:XPN[];
+
+# Union type for holding patient alias name for all supported hl7 versions.
+public type Pid9 hl7v23:XPN[]|hl7v231:XPN[]|hl7v24:XPN[]|hl7v25:XPN[]|hl7v251:XPN[]|hl7v26:XPN[]|hl7v27:ST|hl7v28:ST;
```

Comment:
Should we move these types and the ones in the mappings.bal to a separate file? IMO, it will increase readability.